### PR TITLE
Add instructions for ubuntu 22.04

### PR DESCRIPTION
**Pull request by crohr**
Note: this requires that stable/12 gets the .pkgr.yml changes to enable Ubuntu 22.04. Do not merge before.

### CODE REVIEW

**Comment by crohr** [comment]
I wonder if the edit links are incorrect on docs.openproject.com, since it made me open a PR against `release/12.1` while there is a `release/12.2` branch? Let me know if I should change the base branch.

File: docs/installation-and-operations/installation/packaged/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ The package is available for the following Linux distributions:
 
 | Distribution (64 bits only)                 |
 | ------------------------------------------- |
+| [Ubuntu 22.04 Focal](#ubuntu-2204)          |
```
Comment (wielinde) [inline docs/installation-and-operations/installation/packaged/README.md]: @crohr Is "Focal" still the right name? Here it is called "Jammy Jellyfish" https://releases.ubuntu.com/jammy/

Comment (crohr) [inline docs/installation-and-operations/installation/packaged/README.md]:
```suggestion
| [Ubuntu 22.04 Jammy](#ubuntu-2204)          |
```

Comment (crohr) [reply docs/installation-and-operations/installation/packaged/README.md]: thanks for spotting that

**Comment by crohr** [comment]
Superseded by #11025

**Pull request closed without merging**